techrecord.md

```diff
@@ -77,3 +77,55 @@ All temporal objects should be information stable. All methods that increase or
 
 Negative durations make no sense. Therefore there is a need to have both a `plus` and a `minus` method for date/time arithmetics.
 Only integer units are supported in durations and arithmetic. Expressing fractions requires the use of smaller units.
+
+### The properties on Instant should be getXXX() methods
+
+This comes out of [#139](https://github.com/tc39/proposal-temporal/issues/139). It's actually true that the epochNanoseconds and their ilk aren't really the value of the Instant and that the value of the Instant shouldn't be exposed. There is however a clear relation between an Instant and the epoch, so that `fromEpochNanoseconds()` and `getEpochNanoseconds()` should be the way forward.
+
+### TimeZone objects should be a thing
+
+There have been multiple requests that there is an interface to the time-zone data directly. For that reason we have introduced TimeZone objects. They stringify to the IANA-name of the timezone or the Offset-String in case of a pure offset.
+
+### OffsetDateTime
+
+Since `TimeZone` objects are now a thing and have all the information necessary to distinguish between pure offsets and actual IANA time-zones the distinction between `OffsetDateTime` and `ZonedDateTime` has become meaningless. Therefore we are dropping `OffsetDateTime`.
+
+### Duration strings
+
+`Duration` objects stringify to ISO8601 duration strings and can also be created from such strings. Which makes it easier to work with durations. Any place a duration can be passed into an API either an actual `Duration` object, or a POJO with the same fields, or an ISO8601 duration string can be passed.
```

Comment:
"POJO" is a bit confusing, please replace it with "object" or "plain object".

And also, if we're going to accept time zone strings in place of TimeZone objects in e.g. `withZone` and duration strings in place of Duration objects in e.g. `plus`, then why shouldn't we accept strings in place of _any_ temporal object (e.g., in `withTime`)?

Reply:
Because we have the cast to Duration already. Also TimeZone objects are permanent. They have a === guarantee.

So for those it’s already part of the process. 

Though it’s worth thinking more about it.

Reply:
> Because we have the cast to Duration already. Also TimeZone objects are permanent. They have a === guarantee.

Neither of those seem relevant to this question. To make things concrete, why should some but not all of the following work?
```js
Time.fromString("12:00").plus("PT1H")
DateTime.fromString("2019-09-10T12:00").withZone("America/New_York")
Date.fromString("2019-09-10").withTime("12:00")
```